pkg/utils/config/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	tomlv2 "github.com/pelletier/go-toml/v2"
+)
+
+const (
+	tomlFilePath = "/etc/containerd/config.toml"
+	nriPluginKey = "io.containerd.nri.v1.nri"
+	disableKey   = "disable"
+)
+
+func main() {
+	// Load TOML content
+	tomlData, err := os.ReadFile(tomlFilePath)
+	if err != nil {
+		log.Fatalf("Error reading TOML file: %v", err)
+	}
+
+	tomlMap, err := unmarshalTOML(tomlData)
+	if err != nil {
+		log.Fatalf("Error unmarshalling TOML: %v", err)
+	}
+
+	tomlMap = updateNRIplugin(tomlMap)
+
+	buf, err := addIndentation(tomlMap)
+	if err != nil {
+		log.Fatalf("Error adding indentation: %v", err)
+	}
+
+	if err := writeToFile(tomlFilePath, buf.String()); err != nil {
+		log.Fatalf("Error writing to file: %v", err)
+	}
+
+	if err := runNsenter(); err != nil {
+		log.Fatalf("Failed to restard the containerd: %v", err)
+	}
+}
+
+func runNsenter() error {
```

Comment:
nit: why not to call this `restartContainerd()` ?

Reply:
done